Core/tests/Unit/Batch/SysvConfigStorageTest.php

```diff
@@ -68,11 +68,67 @@ public function testSaveBadConfig()
         } catch (\RuntimeException $e) {
             // verify we didn't corrupt memory
             $this->storage->load();
+            // Just to avoid the test warning
+            $this->assertTrue(true);
             return;
         }
 
         $this->assertTrue(false, 'should have thrown an exception');
     }
+
+    public function testDefaultValues()
+    {
+        if (getenv('GOOGLE_CLOUD_BATCH_SHM_SIZE')
```

Comment:
If we save the original env vars in setup, then you can unset these values with `putenv('VARNAME')` without the `=`.

Reply:
Done